.github/workflows/publish_packages.yml

```diff
@@ -14,7 +14,28 @@ env:
 jobs:
   build-packages:
     uses: ./.github/workflows/build.yml
-  
+
+  build-windows-packages:
+    uses: ./.github/workflows/build-windows.yml
+
+  publish-github:
+    needs: [build-packages, build-windows-packages]
+    if: contains(github.ref , 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/download-artifact@master
+      with:
+        name: linux-amd64-binary
+        path: linux
+    - uses: actions/download-artifact@master
+      with:
+        name: windows-compiled
+        path: windows
+    - uses: ncipollo/release-action@v1
+      with:
```

Comment:
we could consider adding prerelease: true below, to make sure a release is only an actual release after human review.